tests/unit/project_loader/test_config.py

```diff
@@ -2598,6 +2598,68 @@ def test_valid_restart_conditions(self):
         project_loader.Validator(self.data).validate()
 
 
+class ValidStopRefreshModesTestCase(ValidationBaseTestCase):
```

Comment:
Can we split these into separate test cases, please (stop and refresh)?

Reply:
awe, but I was so smart here